ta/crypt/derive_key_taf.c

```diff
@@ -160,7 +160,7 @@ TEE_Result derive_ta_unique_key_test(uint32_t param_types,
 			if (res != TEE_SUCCESS)
 				continue;
 
-			EMSG("Small key test iteration %d failed", i);
+			EMSG("Small key test iteration %ld failed", i);
```

Comment:
The conversion specifier should actually be `zu` since `i` is a `size_t`.

Reply:
Done, thanks for the review!